exercise-src/README.md

```diff
@@ -0,0 +1,66 @@
+# Academy Generator
+
+## File Structure
+
+The source files used to generate the final files are located in `exercise-src`. They are organized by course, module, and project. Generation is handled in alphabetical order, so the modules and projects should be named accordingly, usually with numeric prefixes.
+
+```
+└── <course name>
+    └── <module name>
+        └── <project name>
+```
+
+## Generating
+
+To generate all courses, run
+
+```shell
+npm run generate
+```
+
+To generate a specific course, run
+
+```shell
+npm run generate <course name>
+```
+
+To generate a specific module in a given course, run
+
+```shell
+npm run generate <course name> <module name>
+```
+
+To generate a specific project in a specific module in a specific course, run
+
+```shell
+npm run generate <course name> <module name> <project name>
```

Comment:
Examples for each of these would be great. I don’t really know what a “project” is here.

Reply:
The "File Structure" section above shows the `<course>/<module>/<project>` structure. In React Vite and React Native, the projects are usually `01-problem`, `01-solution`, etc.